### DISCUSSION TITLE
Канал svoddru временно не транслирует новые сообщения

### DISCUSSION THREAD

**Discussion by audetv** [audetv/cohesion#13] [Announcements]
По техническим причинам трансляция сообщений на канал приостановлена.

Все текущие сообщения доступны на страницах сайтов: 
https://фкт-алтай.рф/qa/question/view-7225
https://svodd.ru/bolshaya-svoddnaya-tema?sort=-date

**Comment by audetv** [comment]
Есть ли связь?

> Совещание с членами Совета Безопасности, Правительства и руководством силовых ведомств
> Владимир Путин провёл совещание с членами Совета Безопасности, Правительства и руководством силовых ведомств.
> 30 октября 2023 года 21:15 Московская область, Ново-Огарёво
> http://kremlin.ru/events/president/news/72618

Технические подробности https://github.com/terratensor/tg-svodd-bot/issues/21#issue-1969786557

**Comment by audetv** [comment]
На данным момент на канал созданный для тестирования возможность отправки сообщений восстановлена. 
Сообщения отправляются без ошибок. Ожидаем публикации в соборной теме.

**Comment by audetv** [comment]
Трансляция восстановлена.

**Comment by iprst** [comment]
Странное поведение?

**Reply by audetv** [reply]
Да. так себе, никаких оповещения от телеграм не было. Решил написать сообщение, что не работает трансляция, т.к. не имел представления когда будет восстановлена работоспособность, так то за итоговый период было всего одно сообщение, никто бы и не заметил… Почему не работало не понятно, возможно для тех, кто давно с телеграм это уже обычно и уже случалось, думаю я в следующий раз уже не буду проверять и перевыпускать токены, проверять резервные каналы и ботов, чтобы установить что случилось, забанили, удалили, сломали, взломали…)

**Reply by iprst** [reply]
Вчера была некоторая возня из-за выпуска новой версии клиента, где введены новые функции. Возможно для API ботов тоже какое-то усложнение прилетело. Например в клиенте можно теперь цитировать фразу из комментария для ответа, будет прикреплена только одна эта фраза, а не вся портянка.

**Reply by audetv** [reply]
Не знаю, где об этом почитать) Так то первым делом на сайте тг стал смотреть, но там тишина. Посмотрел отправку сообщений, ну, типа апи поменяли в описании, но не оповестили, но тоже все было по старому. Герметичненько.

**Reply by iprst** [reply]
Да, видимо очередное «даже бабушка моего кота это знает». Пусть так. Пока не критично.

**Reply by iprst** [reply]
А может и собирают ОС по вопросу того, как пользователи разных API будут реагировать на ошибки в работе сервисов после определённых обновлений.

**Reply by audetv** [reply]
Как это выглядит, размышления.

В какой-то момент у всех ботов в сети телеграм пропали права на отправку сообщений в каналы, и публичные и приватные. Была отключена автоматизированная отправка сообщений, возможность ручной отправки сообщений администраторами каналов сохранилась. Таким образом сокращается частота сигнала, который может быть передан в автоматическом режиме различными ботами и бот-сетями. 

Мы не знаем у всех ли ботов были проблемы или выборочно и были исключения, но факт — такая возможность существует, возможность резко понизить частоту сигнала. Кому это может быть нужно и для каких целей? Можно поразмышлять.

Но вывод можно сделать такой, что строить свое управление только на соцсетях и мессенджерах может быть трагично для управления. Надо использовать резервные каналы передачи сигнала, например сайты. Сайты и ПО использующее открытый исходный код, избегать проприетарных движков и фреймворков, чтобы в любой момент можно было провести анализ и разобраться, что делает каждая строка кода.

**Reply by iprst** [reply]
Об этом вопросе надо в теме изложить.
